web/satellite/src/components/header/projectSelection/ProjectSelectionDropdown.vue

```diff
@@ -42,6 +42,7 @@ export default class ProjectSelectionDropdown extends Vue {
 
     public async onProjectSelected(projectID: string): Promise<void> {
         this.$store.dispatch(PROJECTS_ACTIONS.SELECT, projectID);
+        localStorage.setItem('selectedProjectId', this.$store.getters.selectedProject.id);
```

Comment:
lets wrap all localstorage calls in some programming module, not to populate a lot of constants and hardcoded strings